LPavicPlanetIX/qgroundcontrol-PlanetIX
# Virtual terminate button planet ix

Virtual Terminate Button

Description
-----------
Virtual terminate button for immediate termination.
- Image of virtual button displayed on fly view
- Modifying size of confirmation window
- Remove using space bar key for sliding on confirmation window

Test Steps
-----------
- Run QGC and connect the vehicle to it
- Arm the vehicle and fly it
- While the flight lasts, hit the terminate button displayed on fly view
- Confirmation window will pop out on top of the view
- Slide to confirm termination

Checklist:
----------


Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
